src/Components/Blazor/Blazor/ref/Microsoft.AspNetCore.Blazor.csproj

```diff
@@ -7,6 +7,7 @@
     <Compile Include="Microsoft.AspNetCore.Blazor.netstandard2.1.cs" />
     <Reference Include="Mono.WebAssembly.Interop"  />
     <Reference Include="Microsoft.AspNetCore.Components.Web"  />
+    <Reference Include="Microsoft.Extensions.Configuration"  />
```

Comment:
I'll be measuring the impact of this as part of this change. I believe we can also drop options here.